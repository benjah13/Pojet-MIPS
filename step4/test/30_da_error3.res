
	Disassembly of section .text

00000000
      00: 00000000	NOP
Invalid param : maximum number of instructions to display is 1000
