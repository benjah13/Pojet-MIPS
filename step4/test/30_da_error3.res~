
	Disassembly of section .text

00000000
      00: 00000000	NOP

	End of section .text

Invalid param : maximum number of instructions to display is 1000
