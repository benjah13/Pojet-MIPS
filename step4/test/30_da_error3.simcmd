# test da erreur arg2 

da 0x000:1
da 0xae1f:200000000000
