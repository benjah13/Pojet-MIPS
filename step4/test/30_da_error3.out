00000000:	NOP
Invalid param : maximum number of instructions to display is 1000
